File: Java Script - Introduction to Programming/3.1. Simple Conditions/12. Speed Info.js
function speedInfo(n) {
  return n <= 10 ? 'slow' :
    n > 10 && n <= 50 ? 'average' :
    n > 50 && n <= 150 ? 'fast' :
    n > 150 && n <= 1000 ? 'ultra fast' : 'extremely fast'
}

// console.log(speedInfo(19992.9))

File: Pyrhon - Introduction to Programming/3.1. Simple Conditions/12. Speed Info.py
n = float(input())
result = ''

if n <= 10:
  result = 'slow'
elif 10 < n <= 50:
  result = 'average'
elif 50 < n <= 150:
  result = 'fast'
elif 150 < n <= 1000:
  result = 'ultra fast'
else:
  result = 'extremely fast'

print(result)


'''
22
#average
'''

